src/lang/hindi.txt

```diff
@@ -3,8 +3,8 @@
 ##isocode hi_IN
 ##plural 0
 ##textdir ltr
-##digitsep ,
-##digitsepcur ,
+##numberformat 00,000,000,000,000,000,000
```

Comment:
Looks like one of the times when you updated the PR, the translated number format and abbreviations were somehow lost.
```suggestion
##numberformat 0,00,00,00,00,00,00,00,00,000
```

Reply:
I have deliberately reverted that to the original format, because the format and abbreviations should be in-sync with eachother and I do not know how the digit grouping with the abbreviations is. Is it 10,000 crore or 100,00 crore? So I will leave it to the translators.

For CJK it is much clearer and since their translators are active, I have a clearer image what needed to happen there, and when I was wrong or copied the wrong character they quickly notified me.